src/ui/ui-kit/ModalDialogs/BottomSheetDialog/BottomSheetDialog.tsx

```diff
@@ -4,23 +4,35 @@ import type { BaseDialogProps } from '../BaseDialog';
 import { BaseDialog } from '../BaseDialog';
 import * as s from './styles.module.css';
 
+export enum AnimationPreset {
```

Comment:
Do we have any examples of different styles in the extension right now?

Reply:
We're about to ;)